lldb/tools/lldb-dap/DAP.cpp

```diff
@@ -962,6 +962,44 @@ bool ReplModeRequestHandler::DoExecute(lldb::SBDebugger debugger,
   return true;
 }
 
+// Sends a custom DAP event with an optional body.
+//
+// See
+// https://code.visualstudio.com/api/references/vscode-api#debug.onDidReceiveDebugSessionCustomEvent
+bool CustomDAPEventRequestHandler::DoExecute(
+    lldb::SBDebugger debugger, char **command,
+    lldb::SBCommandReturnObject &result) {
+  // Command format like: `custom-event <name> <body>?`
```

Comment:
afaict, we currently do not reject

```
custom-event foo {} bar baz
```

and simply ignore `bar` and `baz`? We should probably raise an error instead?

Reply:
I added a check for additional arguments and returned an error if to many are found.

Reply:
I couldn't find where we are handling this in the code. Did you already update the PR with those changes?

Reply:
Ah, I found it now. Looked a bit different than what I had in mind, and hence my simple scanning for `size()` didn't find it. See other comment